cg/services/delivery_message/messages/analysis_scout_message.py

```diff
@@ -1,16 +1,20 @@
 from cg.services.delivery_message.messages.delivery_message import DeliveryMessage
-from cg.services.delivery_message.messages.utils import get_fastq_delivery_path, get_scout_link
+from cg.services.delivery_message.messages.utils import (
+    get_fastq_delivery_path,
+    get_scout_link,
+)
 from cg.store.models import Case
 
 
 class AnalysisScoutMessage(DeliveryMessage):
-    def create_message(self, case: Case) -> str:
-        scout_link: str = get_scout_link(case)
-        delivery_path: str = get_fastq_delivery_path(case)
+    def create_message(self, cases: list[Case]) -> str:
+        scout_links: list[str] = [get_scout_link(case) for case in cases]
+        scout_links_row_separated: str = "\n".join(scout_links)
+        delivery_path: str = get_fastq_delivery_path(cases[0])
         return (
             f"Hello,\n\n "
-            f"The results have been uploaded to Scout for the following case:\n\n"
-            f"{scout_link}\n\n"
+            f"The results have been uploaded to Scout for the following cases:\n\n"
```

Comment:
```suggestion
            f"The results have been uploaded to Scout for the following case(s):\n\n"
```
If len(cases) == 1

Reply:
I have just modeled it from the SuSy templates currently.

Reply:
it is about the grammar, one case two or more cases, hence case(s). I think this is cleaner than using if else. Sorry I should have been more explicit.

Reply:
@beatrizsavinhas has spoken. No s at all if one case, always an s otherwise